kong/clustering/rpc/socket.lua

```diff
@@ -152,7 +152,13 @@ function _M:process_rpc_msg(payload, collection)
     ngx_log(ngx_DEBUG, "[rpc] got RPC call: ", payload_method, " (id: ", payload_id, ")")
 
     local dispatch_cb = self.manager.callbacks.callbacks[payload_method]
-    if not dispatch_cb and payload_id then
+    if not dispatch_cb then
+      -- for RPC notify
+      if not payload_id then
+        return nil, "unable to find RPC notify callback for method: " .. payload_method
+      end
```

Comment:
Shall we consider it as an error? I think it should be ok for which notify rpc call has no handler.

Reply:
Yes, it should not be an error, we could log it then continue.

Reply:
@chobits , could you change it? thanks.